src/EditorFeatures/Core/InheritanceMargin/InheritanceMarginServiceHelpers.cs

```diff
@@ -194,89 +220,168 @@ internal static class InheritanceMarginServiceHelper
             var baseSymbolItems = await baseSymbols
                 .SelectAsArray(symbol => symbol.OriginalDefinition)
                 .Distinct()
-                .SelectAsArrayAsync((symbol, _) => CreateInheritanceItemAsync(solution, symbol, InheritanceRelationship.Implementing, cancellationToken), cancellationToken)
+                .SelectAsArrayAsync((symbol, _) => CreateInheritanceItemAsync(
+                    solution,
+                    symbol,
+                    InheritanceRelationship.InheritedInterface,
+                    cancellationToken), cancellationToken)
                 .ConfigureAwait(false);
 
             var derivedTypeItems = await derivedTypesSymbols
                 .SelectAsArray(symbol => symbol.OriginalDefinition)
                 .Distinct()
-                .SelectAsArrayAsync((symbol, _) => CreateInheritanceItemAsync(solution, symbol, InheritanceRelationship.Implemented, cancellationToken), cancellationToken)
+                .SelectAsArrayAsync((symbol, _) =>
+                    CreateInheritanceItemAsync(solution,
+                    symbol,
+                    InheritanceRelationship.ImplementingType,
+                    cancellationToken), cancellationToken)
```

Comment:
indentation is wonky here.  also code is inconsistently wrapped with similar pattern above.  i.e. CreateInheritanceItemAsync is not wrapped in teh first form, but is in the second.